conf/setup.conf

```diff
@@ -14,11 +14,6 @@ TARGET-ARCH-FLAGS = -DM32
 TARGET-ARCH	= riscv32-elf
 
 
-#
-#       Specifies the architecture used. Uncomment for riscv builds.
-#
-#ADDITIONAL_ARCH_FLAGS = --with-arch=rv32ifd
-
```

Comment:
In 7a05e29c5a77906fbbace4691dc12e5a8793b9c5 @phillipstanleymarbell removed these lines from  `tools/Makefile`. I assume that this PR should not add them back in?

Reply:
I think so.